src/scripts/components/stories/story/blocks/image-gallery/formats/scroll-overlay/scroll-overlay-slide/scroll-overlay-slide.module.css

```diff
@@ -0,0 +1,72 @@
+.slide {
+  scroll-snap-align: center;
+  overflow: hidden;
+  position: relative;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  color: white;
+
+  /* Either video or image as background */
+  .assetContainer {
+    position: absolute;
+    top: 0;
+    left: 0;
+    width: 100%;
+    height: 100%;
+    z-index: 1;
+    will-change: transform;
+  }
+
+  .asset {
+    width: 100%;
+    height: 100%;
+    object-fit: cover;
+  }
+
+  &:not(.first):before {
```

Comment:
There is a selector for the first. https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child